packages/block-ingestor/db/migrations/0001_initial_schema.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE blocks (
+  hash TEXT NOT NULL,
+  height INTEGER NOT NULL,
+  network TEXT NOT NULL,
+  kv_key TEXT NOT NULL,
+  status TEXT NOT NULL DEFAULT 'new',
+  inserted_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
```

Comment:
I would prefer if we have everything in miliseconds, so we don't need to have default, and keep `NOT NULL`